test/compflags/TestWarnUnstableSuppression.7.good

```diff
@@ -1,4 +1,4 @@
 TestWarnUnstableSuppression.chpl:2: warning: chpl_unstableInternalSymbolForTesting is unstable
 TestWarnUnstableSuppression.chpl:3: warning: chpl_unstableStandardSymbolForTesting is unstable
-$CHPL_HOME/modules/internal/ChapelBase.chpl:nnnn: warning: chpl_unstableInternalSymbolForTesting is unstable
 $CHPL_HOME/modules/standard/Reflection.chpl:nnnn: warning: chpl_unstableStandardSymbolForTesting is unstable
```

Comment:
It moved later, actually. I did not investigate the reason this order changed because I view it as irrelevant. The warning output from the compiler is not something we are stabilizing. Also, we have many production compiler passes that work in an order that depends on ordering in the compiler's internal representations, and even in dyno warning ordering will depend on the order in which the modules were processed. This PR changes both the order in which dyno initially processes various modules and the resulting ordering in the converted AST. So this kind of change is not surprising or worrying to me.